addons/l10n_lt/i18n/lt.po

```diff
@@ -0,0 +1,1713 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#   * l10n_lt
```

Comment:
Could you please move translation files(.po/.pot) to new i18n_extra directory ?
Generally, translation files for l10n modules should be placed under i18n_extra directory instead of placing it under i18n since these terms are to be manually translated.

Also, there isn't an empty translation template file(l10n_it.pot) for your module, can you please generate(using Settings/Translation/Export Translation menu) and place it in i18n_extra directory well ?